frontend/ui/elements/page_turns.lua

```diff
@@ -78,4 +97,48 @@ if Device:hasKeys() then
         end,
     })
 end
+
+local function genTapZonesMenu(tap_zones_type)
+    table.insert(PageTurns.sub_item_table[3].sub_item_table, {
```

Comment:
Always a bit risky of future mess up to reference `PageTurns.sub_item_table[3]` here and below.
You could just create it as a `local page_turns_tap_zones_sub_items =` and on line 45 use `sub_item_table = page_turns_tap_zones_sub_items`.